scripts/prepare_sector_network.py

```diff
@@ -747,7 +748,7 @@ def add_allam(n, costs):
         p_nom_extendable=True,
         # TODO: add costs to technology-data
         capital_cost=0.6 * 1.5e6 * 0.1,  # efficiency * EUR/MW * annuity
-        marginal_cost=2,
```

Comment:
Drawing from the gas bus already covers the gas fuel cost. In general, power plants implemented as links do not include fuel cost in their `marginal_cost`. The 2 €/MWhth is only the VOM.

Reply:
Makes perfect sense. Thank you !